extras/conda-nest-dev.yml

```diff
@@ -0,0 +1,106 @@
+# Conda environment specification for NEST Simulator
```

Comment:
I suggest to rename this file, the contained environment and all references to those to `nest-simulator-dev` in order to avoid any overlap with software for the infamous thermostat. Better be safe than sorry.